### DISCUSSION TITLE
Unnecessary package drawer ,only gives error !! just fed up

### DISCUSSION THREAD

**Discussion by joy-mollick** [react-navigation/react-navigation#10075] [Q&A]
I just created bottom tabs in my app perfectly but for another reason I tried to use navigation drawer . But whole project causes error only . Reack hooks problem ,even I removed implementation of drawer , still gets error . Just a bad package ever I used from react-navigation . You should maintain this or close this package as there are so many issues !!! There is no clear documentation also to pull these errors off !!!

**Comment by satya164** [comment]
wow that was such a helpful comment. totally helped me to figure out what the issue was. I have never seen so helpful post with such a detailed description of the errors.
